src/website/views/templates/order-checkout-flow.pug

```diff
@@ -0,0 +1,1697 @@
+                Tim Anderson
+              </button>
+              <div class="m-dropdown__menu" style="width:10rem;">
+                <a class="m-dropdown__menu-item -active" href="#">
+                  link 1</a>
+                <a class="m-dropdown__menu-item" href="#">
+                  link 2
+                </a>
+                <a class="m-dropdown__menu-item" href="#">
+                  link 3
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </header>
+      <div class="a-divider -inverse -bg--black -m--0"></div>
+      <div class="-bg--black">
+        <ul class="a-steps -horizontalLabels -inverse -py--2 -m--0 -pr--4 -mw--640 -m--auto">
+          <li class="-completed">
+            <div class="a-steps__icon"></div>
+            <a href="#">Customer information</a>
+            <div class="a-steps__line"></div>
+          </li>
+          <li class="-active">
+            <div class="a-steps__icon"></div>
+            <a href="#">Configure locations</a>
+            <div class="a-steps__line"></div>
+          </li>
+          <li>
+            <div class="a-steps__icon"></div>
+            <a href="#">Checkout</a>
+          </li>
+        </ul>
+      </div>
+      <div class="-d--flex">
+        <div class="-br--1 -d--flex -flex--column -justify-content--between" style="min-width:200px">
+          <div>
+            <div class="-d--flex -flex--row -justify-content--start -pl--3 -py--1">
+              <h5>Locations</h5>
+              <span class="-d--none -my--1 -ml--1" id="locations">(1)</span>
+            </div>
+            <ul class="a-tabs -vertical -large">
+              <li class="-active -bg--grey-20" id="locationActiveTab">
+                <a href="#">New Location</a>
+              </li>
+              <li id="locationTab" style="display:none">
+                <a class="-text--larger" href="#">
+                  <i class="a-icon icon-map-marker -mr--1"></i>
+                  CenturyLink
+                </a>
+                <div class="-pl--6">
+                  <p class="-text--small -mt--0">
+                    1100 112th Avenue <br>
+                    NE, Suite 400<br>
+                    Bellevue, WA 98004
+                  </p>
+                </div>
+              </li>
+            </ul>
+          </div>
+          <div class="-p--1 -d--flex -flex--row -justify-content--start -align-items--center -bt--1">
+            <i class="a-icon icon-circle-plus-outline -sm3 -text--brand -ml--1"></i>
+            <h6 class="-text--brand -ml--1">
+              Add Location</h6>
+          </div>
+        </div>
+        <div class="-px--5 -py--2 -m--auto -flex--grow1" style="height:800px;overflow:scroll;" id="checkout">
+          <!--Location Step-->
+          <div class="m-epanel -active" data-chi-epanel-group="checkout">
+            <div class="m-epanel__header">
+              <span class="m-epanel__number">1</span>
```

Comment:
`m-epanel__number` should be styled the same as `m-epanel__title`. Same semi-bold font-weight, same brand color when active, etc.

Reply:
done